### ISSUE TITLE
debian: combine changelogs?

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#1126]
Since switching to be a native package, we now have 2 changelogs:
- https://github.com/freedombox/Plinth/blob/master/CHANGELOG.md
- https://github.com/freedombox/Plinth/blob/master/debian/changelog

Should these be combined?

**Comment by SunilMohanAdapa** [comment]
Let us combine our changelogs.  There is no reason to keep two of them.

**Comment by JoKeyser** [comment]
What should be the format and location of the combined file? E.g. `debian/changelog` mentions uploaders and Debian tags(?) like unstable, urgency... Should that be kept or not?

**Comment by jvalleroy** [comment]
Well, we have to keep debian/changelog, with the correct format. That is required to build the package.

**Comment by JoKeyser** [comment]
So... it boils down to erasing CHANGELOG.md?

**Comment by jvalleroy** [comment]
Yes, but the entries that were added to CHANGELOG.md before, should now be added to debian/changelog instead.

Something like this:
```
[Contributor 1]
* Change 1

[Contributor 2]
* Change 2
* Change 3
```

**Comment by SunilMohanAdapa** [comment]
@jvalleroy , I think instead of removing the old changelog and trying to move it's contents to debian/changelog, we can keep the old one and write an at the top mentioning that new change log entries will go into debian/changelog. Changelogs entries are usually not modified after entries are made.

**Comment by JosephKiranBabu** [comment]
@SunilMohanAdapa We currently do not have a CHANGELOG.md file.

**Comment by SunilMohanAdapa** [comment]
@JosephKiranBabu , My suggestion is to resurrect it and add the line.
